Automattic/jetpack
# Dynamically remove and append rows in the forms dashboard

## Proposed changes:

This PR updates the forms dashboard to dynamically remove and append rows when using bulk actions, instead of fully refreshing the list every time.

### Other information:


## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






- Go to `admin.php?page=jetpack-forms`.
- You should notice the new loading placeholders on the table.
- Select some responses and perform a bulk action.
- The selected rows should disappear immediately while the rest remains unaffected.
- You should see a loading placeholder appear at the bottom of the list and it should eventually append any new rows that now fit on the page if applicable.
